src/models/Organization.ts

```diff
@@ -46,8 +61,16 @@ export function init(): void {
 
 export function createAssociations(): void {
   Organization.belongsToMany(User, {
+    as: "users",
     foreignKey: { name: "ipaCode", field: "organizationIpaCode" },
     otherKey: { name: "fiscalCode", field: "userFiscalCode" },
     through: OrganizationUser
   });
+  Organization.belongsTo(User, {
+    as: "legalRepresentative",
+    foreignKey: {
+      field: "userFiscalCode",
```

Comment:
let's name the fields `legalRepresentativeFiscalCode` instead of a generic `userFiscalCode`